examples/model_selection/plot_roc.py

```diff
@@ -101,7 +124,11 @@
 
 
 ##############################################################################
-# Plot ROC curves for the multiclass problem
+# Plot ROC curves for the multiclass problem using One vs. Rest classification.
+
+# Compute micro-average ROC curve and ROC area
+fpr["micro"], tpr["micro"], _ = roc_curve(y_test.ravel(), y_score.ravel())
```

Comment:
Good point. @amueller, do you have input on this? Correct me if I'm wrong, but I interpreted this comment as "should we add a `average="micro"` parameter to the [roc_curve function](https://github.com/scikit-learn/scikit-learn/blob/dc4255f383b9add94b0afb24dbcc39108fa2c09f/sklearn/metrics/ranking.py#L424)?" The micro-average was in the original `plot_roc.py` example already -  I've just moved it so that it would be in the One vs. Rest section of the expanded example, which is why it shows up in the diff.